web/templates/compose.hbs

```diff
@@ -108,10 +108,16 @@
                             <div id="below-compose-content">
                                 <div class="compose_bottom_top_container">
                                     <div class="compose_right_float_container order-3">
-                                        <button type="submit" id="compose-send-button" class="button small send_message animated-purple-button" title="{{t 'Send' }} (Ctrl + Enter)">
+                                        <button type="submit" id="compose-send-button" class="button small send_message animated-purple-button" title="{{t 'Send' }} (Ctrl + Enter)" tabindex=0>
                                             <img class="loader" alt="" src="" />
                                             <span>{{t 'Send' }}</span>
                                         </button>
+                                        <button id="schedule-confirm" class="button small hide animated-purple-button tippy-zulip-tooltip" tabindex=0>
+                                            <span>{{t 'Schedule' }}</span>
+                                        </button>
```

Comment:
Should this have a `loader` element inside?

Reply:
Yes, but looking at implementing it, we should do it as a followup since we have to make sure we are showing and hiding the loading indicator for the right element.

Reply:
OK